### PR TITLE
test against the devel branch

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

File: nimble.nimble [hunk 1]
```diff
@@ -22,4 +22,4 @@ when defined(nimdistros):
 
 task test, "Run the Nimble tester!":
   withDir "tests":
-    exec "nim c -r tester"
+    exec "nim c -r -d:useMalloc tester"
```
Comment (ringabout) [inline nimble.nimble:25]:
```suggestion
    exec "nim c -r --gc:refc tester"
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,14 +13,14 @@ jobs:
           - macos-latest
           - ubuntu-latest
         nimversion:
```
Comment (ringabout) [inline .github/workflows/test.yml:15]:
```suggestion
        version:
```

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -12,15 +12,15 @@ jobs:
           - windows-latest
           - macos-latest
           - ubuntu-latest
-        nimversion:
-          - nightly:https://github.com/nim-lang/nightlies/releases/tag/2021-07-18-devel-923a1c6ea7d9f45b6389680717692690218228fb
+        version:
```
Comment (ringabout) [inline .github/workflows/test.yml:15]:
```suggestion
        nimversion:
```

File: tests/tmisctests.nim [hunk 4]
```diff
@@ -84,11 +84,11 @@ suite "misc tests":
       const warningsToCheck = [
         "[UnusedImport]",
         "[DuplicateModuleImport]",
-        "[Deprecated]",
+        # "[Deprecated]", # todo fixme
```
Comment (ringabout) [inline tests/tmisctests.nim:87]: There are some `nimPreviewSlimSystem` deprecation messages which can be fixed later.

Comment (c-blake) [reply tests/tmisctests.nim:87]: Right now things seem to outright fail if the user has `nimPreviewSlimSystem` activated in their user config.. https://github.com/nim-lang/nimble/issues/1050

**Pull request merged**